feature/experimental/leader_failover_test/leader_failover_test.go

```diff
@@ -47,13 +47,13 @@ func TestMain(m *testing.M) {
 
 const (
 	ipv4PrefixLen = 30
-	instance      = "DEFAULT"
 	ateDstNetCIDR = "203.0.113.0/24"
 	nhIndex       = 1
 	nhgIndex      = 42
 )
 
 var (
+	instance = *deviations.DefaultNetworkInstance
```

Comment:
This assignment is too early. At initialization time, the flags have not been parsed yet. I'd suggest just use `*deviations.DefaultNetworkInstance` directly.